src/main/java/elucent/albedo/lighting/LightManager.java

```diff
@@ -74,16 +76,38 @@ public static void update(World world) {
 			if (e instanceof ILightProvider){
 				addLight(((ILightProvider)e).provideLight());
 			}
+			if (e.hasCapability(Albedo.LIGHT_CAP, null)){
```

Comment:
Do you have any idea how worse in performance this can be compared to a simple instanceof across thousands of entities and TEs? This has already been discussed in #3.